Infrastructure.md

```diff
@@ -0,0 +1,58 @@
+# Infrastructure Setup
+
+The provisioning system depends on a central master, hosted at http://classroom.puppet.com
```

Comment:
The idea being that with a central master we have visibility via reports. That will let us know what's failing, what requires a second run, etc. before instructors even report them. It also lets us deploy updates using a standard control-repo workflow and pin to module versions, etc.